parl/core/fluid/layers/tests/param_name_test.py

```diff
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from paddle import fluid
 import parl.core.fluid.layers as layers
```

Comment:
can we use `from parl import layers` here?

Reply:
done